{{ cookiecutter.project_slug }}/Dockerfile

```diff
@@ -0,0 +1,64 @@
+#
+# Base image with defaults for all stages
+FROM registry.access.redhat.com/ubi9/python-{{ cookiecutter.python_version.replace(".", "") }} AS base
```

Comment:
`python_version` is an array in `cookiecutter.json`. Which version is it used?

Reply:
Cookiecutter asks for it. If you use `cookiecutter --no-input` it's the first entry

Reply:
```$ uv tool run cookiecutter  --output-dir /tmp/foobar .

  [1/9] full_name (AppSRE):
  [2/9] email (sd-app-sre@redhat.com):
  [3/9] project_name (Project Name):
  [4/9] project_slug (project-name):
  [5/9] project_module (project_name):
  [6/9] project_short_description (Please enter a short description of your project):
  [7/9] Select deployment
    1 - openshift
    2 - pypi
    Choose from [1/2] (1):
  [8/9] version (0.1.0):
  [9/9] Select python_version
    1 - 3.12
    2 - 3.11
    Choose from [1/2] (1):
```

Reply:
See [66432c3](https://github.com/app-sre/python-project-template/pull/3/commits/66432c332ece65fc75960c69c1cf0e9efc9a0ebe)

Reply:
nice! thx